Dockerfile

```diff
@@ -6,6 +6,15 @@ RUN chmod 755 /bin/ttn-lw-stack
 COPY ttn-lw-cli /bin/ttn-lw-cli
 RUN chmod 755 /bin/ttn-lw-cli
 
+COPY data/lorawan-devices /srv/ttn-lorawan/lorawan-devices
+RUN rm -rf /srv/ttn-lorawan/lorawan-devices/.git
```

Comment:
This should also be avoided. The `lorawan-devices` repo contains lots of image assets that we do want to bundle in release images.

Reply:
Do want or do not want?

This just deletes the Git history, which we don't need in release images.

Reply:
Do not add `lorawan-devices` in the Docker image at all. Copying only the required assets and initializing the index is handled in #3543